examples/grpc-server-with-prometheus/client/client.go

```diff
@@ -3,32 +3,83 @@ package main
 import (
 	"fmt"
 	"log"
+	"net/http"
 
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
+	"github.com/grpc-ecosystem/go-grpc-prometheus"
 
 	pb "github.com/grpc-ecosystem/go-grpc-prometheus/examples/grpc-server-with-prometheus/protobuf"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"os"
+	"os/signal"
+	"syscall"
 )
 
+var (
+	// Create a metrics registry.
+	reg = prometheus.NewRegistry()
+
+	// Create some standard server metrics.
+	grpcMetrics = grpc_prometheus.NewClientMetrics()
+
+	// Create a customized counter metric.
+	customizedCounterMetric = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "demo_client_say_hello_method_handle_count",
+		Help: "Total number of RPCs call from the client.",
+	}, []string{"ClientName1"})
+)
+
+func init() {
+	// Register standard server metrics and customized metrics to registry.
+	reg.MustRegister(grpcMetrics, customizedCounterMetric)
+	customizedCounterMetric.WithLabelValues("ClientTest1")
+}
 func main() {
 	// Create a insecure gRPC channel to communicate with the server.
 	conn, err := grpc.Dial(
 		fmt.Sprintf("localhost:%v", 9093),
 		grpc.WithInsecure(),
+		grpc.WithUnaryInterceptor(grpcMetrics.UnaryClientInterceptor()),
 	)
 	if err != nil {
 		log.Fatal(err)
 	}
 
+	// Create a HTTP server for prometheus.
+	httpServer := &http.Server{Handler: promhttp.HandlerFor(reg, promhttp.HandlerOpts{}), Addr: fmt.Sprintf("0.0.0.0:%d", 9094)}
+
+
```

Comment:
Delete needless space

Reply:
And you should write codes about creating a HTTP server under the `defer conn.Close()`